### DISCUSSION TITLE
Question about Credit Note xml

### DISCUSSION THREAD

**Discussion by djbox3** [stephanstapel/ZUGFeRD-csharp#527] [Q&A]
I attempted to generate a Credit Note and observed the following discrepancies:
**Correct me if I'm wrong.

The generated code includes cbc:InvoiceTypeCode, cac:InvoiceLine, and cbc:LineExtensionAmount.
However, it should be:

cbc:InvoiceTypeCode → cbc:CreditNoteTypeCode
cac:InvoiceLine → cac:CreditNoteLine
cbc:LineExtensionAmount → cbc:CreditedQuantity

**Comment by stephanstapel** [comment]
Yes, you are right. Can you fix it and send a PR?
